content/source/docs/enterprise/upgrade/differences.html.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: enterprise2
+page_title: "Current vs. Legacy - Upgrading - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-upgrading-differences"
+---
+
+# Differences Between Current and Legacy Terraform Enterprise
+
+Terraform Enterprise began as part of the discontinued Atlas suite, and today's version is a ground-up redesign. If you've previously used the legacy version of Terraform Enterprise, we hope you enjoy the new version's improvements.
+
+Several things are significantly different in the new Terraform Enterprise. This page is a brief summary of what to expect when upgrading from the legacy version.
+
+## Renamed or Moved Features
+
+### Workspaces Replace Environments
+
+Environments are now called [workspaces](../workspaces/index.html). Our goal in this change was to make it easier to on-board new Terraform users in your organization by avoiding an overloaded and ambiguous term.
+
+### Event Log is Now Audit Log (Private Installs Only)
+
+[audit]: ../private/logging.html#audit-logs
+
+Legacy Terraform Enterprise included an event log that logged details about run events. For the new Terraform Enterprise, we replaced it with a [comprehensive audit logging system][audit] that also logs all changes to organization and workspace settings.
+
+The new audit log is available on [private installs](../private/index.html) of Terraform Enterprise.
+
+### `ATLAS_CONFIRM_DESTROY` is now `CONFIRM_DESTROY`
+
+In order to queue a destroy plan on an environment (legacy) or workspace (current), you have to confirm your intentions by setting a special environment variable. The name of that variable has changed to `CONFIRM_DESTROY`, as part of our move away from the Atlas brand name.
+
+## Gone But Returning Soon
+
+### Two-Factor Authentication
+
+The new version of Terraform Enterprise doesn't yet support two-factor authentication in its integrated login system. We plan to add it in a future update.
+
+Depending on your security infrastructure, our [SAML single sign-on support](../saml/index.html) (available on private installs) might also suit your needs.
+
+### Notifications
+
+The new Terraform Enterprise doesn't yet support arbitrary notifications at the end of runs. We plan to add a webhook-based notification system in a future update.
+
+## Permanently Removed
+
+We deliberately removed these features to streamline and improve Terraform Enterprise's core features. Some of these features worked against or distracted from what Terraform Enterprise does best, and some are good ideas that are handled better by external services.
```

Comment:
Terraform Enterprise's core _workflow_

Reply:
👍🏼

Reply:
Would recommend adding another paragraph below here again empathizing with the users who are affected by this. "We understand that removing features can be painful and disappointing for those who have built processes around them. For each of these features, we ensured only a single digit percentage of users would be affected, and have attempted to mitigate the change with guides to other systems when appropriate. If you're being majorly affected by these removals and the documentation we've created isn't adequate, please contact us and we'll work with you to help you migrate."